Documentation/admission-controller-usage.md

```diff
@@ -0,0 +1,80 @@
+# Admission Controller
+
+## Requirements
```

Comment:
Let's remove the Requirements section. I think this info will be useful later in the doc. I'll comment separately on that.

Reply:
Removed